### DISCUSSION TITLE
您能给小白出一个客户端使用教程吗？

### DISCUSSION THREAD

**Discussion by jieo66** [apernet/hysteria#1020] [General]
您能给小白出一个客户端使用教程吗？目前很多youtube的教程都是使用v2ray，谢谢大佬，麻烦了

**Comment by haruue** [comment]
不良林出过 Hysteria 2 教程
https://www.youtube.com/watch?v=CXj-ID33MhU

**Comment by jieo66** [comment]
Dear teachers,
I hope this message finds you well.
I am writing to inquire about the router resource file update in version 1.52 of the app. I have noticed that this feature seems to be unavailable in the latest version. Could you please clarify if this is indeed the case and explain the reasons behind this change? I would also appreciate any information on the potential impact of this change on users.
Thank you for your time and attention to this matter. I look forward to your response.
Sincerely,

**Reply by haruue** [reply]
您好， 您可以直接用中文发问， 不必找 ChatGPT 转换成英语。 可以节省大家的时间。
